CMakeLists.txt

```diff
@@ -1,14 +1,17 @@
-cmake_minimum_required(VERSION 3.4)
-
-# stop in source builds
-set(CMAKE_DISABLE_IN_SOURCE_BUILD ON)
+cmake_minimum_required(VERSION 3.7)
+project(inexor-vulkan-renderer)
 
-# options
-option(INEXOR_BUILD_DOC "build documentation" OFF)
+list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake)
 
-message(STATUS "INEXOR_BUILD_DOC = ${INEXOR_BUILD_DOC}")
+# Stop in source builds
+set(CMAKE_DISABLE_IN_SOURCE_BUILD ON)
```

Comment:
Should stay at the very top directly below the minimum required version, because either it stops or not.